### ISSUE TITLE
TypeScript Compilation Failure When Configuring Vitest on WXT

### ISSUE THREAD

**Issue by asasas234** [wxt-dev/wxt#334]
<img width="761" alt="image" src="https://github.com/wxt-dev/wxt/assets/4795442/0ca91891-ee5b-4ec3-9907-096c6b37a4ff">
Hello,

I've encountered an issue while trying to configure Vitest for my WXT project, following the official documentation.

Environment Details:

WXT version: 0.13.4
Vitest version: 1.1.3
TypeScript version: 5.3.3
Issue Description:
I've added the WxtVitest plugin to the vitest.config.ts as instructed, but TypeScript is failing to compile. The error message suggests that there's no overload that matches the plugin call.

**Comment by aklinker1** [comment]
Please provide a minimal reproduction.

**Comment by asasas234** [comment]
@aklinker1 I have resolved the issue. I found that the problem was with the version of "vitest" that I was using. Vite automatically pulls in the version, which was 5.0.11, and the version of "vitest" I previously used was 1.1.3, which depended on Vite 5.0.10. I changed it to 1.1.0, which is the same version as the example in "wxt-example", and it worked fine. Otherwise, in WebStorm, TypeScript would show compilation issues with type mismatch, even though there were no actual runtime problems, it just didn't look good.
